src/services/projectService.ts

```diff
@@ -41,6 +42,18 @@ class ProjectService {
     return pncClient.getHttpClient().post('/projects', data, requestConfig);
   }
 
+  /**
+   * Patch a Project.
+   *
+   * @param id - project ID
+   * @param patchData - array of changes in JSON patch format
```

Comment:
Maybe we could keep naming consistent with the official documentation https://www.npmjs.com/package/fast-json-patch , so:
`... in JSON-Patch format`